share/doc/homebrew/Acceptable-Formulae.md

```diff
@@ -66,9 +66,8 @@ Formulae in the core repository must have a stable version tagged by
 the upstream project. Tarballs are preferred to git checkouts, and
 tarballs should include the version in the filename whenever possible.
 
-Software that does not provide a stable, tagged version, or had guidance to
-always install the most recent version, should be put in
-[homebrew/head-only](https://github.com/Homebrew/homebrew-head-only) or [homebrew/devel-only](https://github.com/Homebrew/homebrew-devel-only).
+Software that does not provide a stable, tagged version should be put in
```

Comment:
What about this?

> Software that only provides a development/beta, tagged version should be put in
> devel-only. We don’t accept software without a tagged version because it
> provides a terrible UX and quickly becomes outdated.